docs/en/serverless/logging/troubleshoot-logs.mdx

```diff
@@ -20,7 +20,7 @@ if you don't have the required privileges to create an API key, you'll see the f
 
 You need to either:
 
-* Ask an administrator to update your user role to at least **Deployment access** → **Admin**. Read more about user roles in <DocLink slug="/serverless/general/assign-user-roles" />. After your use role is updated, restart the onboarding flow.
+* Ask an administrator to update your user role to at least **Deployment access** → **Admin**. Read more about user roles in <DocLink slug="/serverless/general/assign-user-roles">Assign user roles and privileges</DocLink>. After your use role is updated, restart the onboarding flow.
```

Comment:
Is hard coding the link text really required? I just think this is a maintainability issue. (IMO it's better for the tool to keep the link text up-to-date unless there is a compelling reason to hard code it.)

I haven't looked at the original output so maybe this does solve a problem, but I've used the syntax without the link text in the past, and it's worked.

Reply:
This was specifically for migrating to AsciiDoc -- it was complicated to get the link text for a page in a different repo -- but I can put some more time into the script if this is a deal breaker.

Reply:
```suggestion
* Ask an administrator to update your user role to at least **Deployment access** → **Admin**. Read more about user roles in <DocLink slug="/serverless/general/assign-user-roles" />. After your use role is updated, restart the onboarding flow.
```